evap/staff/views.py

```diff
@@ -347,16 +351,64 @@ def semester_export(request, semester_id):
             if 'selected_course_types' in form.cleaned_data:
                 course_types_list.append(form.cleaned_data['selected_course_types'])
 
-        filename = "Evaluation-%s-%s.xls" % (semester.name, get_language())
+        filename = "Evaluation-{}-{}.xls".format(semester.name, get_language())
         response = HttpResponse(content_type="application/vnd.ms-excel")
-        response["Content-Disposition"] = "attachment; filename=\"%s\"" % filename
+        response["Content-Disposition"] = "attachment; filename=\"{}\"".format(filename)
         ExcelExporter(semester).export(response, course_types_list, ignore_not_enough_answers, include_unpublished)
         return response
     else:
         return render(request, "staff_semester_export.html", dict(semester=semester, formset=formset))
 
 
 @staff_required
+def semester_raw_export(request, semester_id):
+    semester = get_object_or_404(Semester, id=semester_id)
+
+    filename = "Evaluation-{}-{}_raw.csv".format(semester.name, get_language())
+    response = HttpResponse(content_type="text/csv")
+    response["Content-Disposition"] = "attachment; filename=\"{}\"".format(filename)
+
+    writer = csv.writer(response, delimiter=";")
+    writer.writerow([_('Name'), _('Degrees'), _('Type'), _('Single result'), _('State'), _('#Voters'),
+        _('#Participants'), _('#Comments'), _('Average degree')])
+    for course in semester.course_set.all():
+        degrees = ", ".join([degree.name for degree in course.degrees.all()])
+        course.avg_grade, course.avg_deviation = calculate_average_grades_and_deviation(course)
+        if course.state in ['evaluated', 'reviewed', 'published'] and course.avg_grade is not None:
+            avg_grade = "{:.1f}".format(course.avg_grade)
+        else:
+            avg_grade = ""
+        writer.writerow([course.name, degrees, course.type.name, course.is_single_result(), course.state,
+            course.num_voters, course.num_participants, course.textanswer_set.count(), avg_grade])
+
+    return response
+
+
+@staff_required
+def semester_participation_export(request, semester_id):
+    semester = get_object_or_404(Semester, id=semester_id)
+    participants = [participant for participant in UserProfile.objects.filter(courses_participating_in__semester=semester).order_by("username")]
```

Comment:
you skipped the `can_user_use_reward_points`, intentionally? if you add it, please put the list comprehension on a second line